5_databases/week2/app.js

```diff
@@ -0,0 +1,39 @@
+import express from "express";
+import { readFileSync } from "node:fs";
+
+const app = express();
+const port = process.env.PORT || 3003;
+
+
+const data = readFileSync("./documents.json", (err, data) => {
+  if (err) throw err;
+});
+const documents = JSON.parse(data);
+console.log(documents);
+
+
+// Support parsing JSON requests
+app.use(express.json());
+
+app.get("/", (req, res) => {
+  res.send("This is a search engine");
+});
+
+app.get("/search", (req, res) => {
```

Comment:
Where is the rest :sweat_smile: 
![Alt Text](https://media.giphy.com/media/v1.Y2lkPTc5MGI3NjExYjlpcTg4NGt4bW9qb3o5dDA1aWh4aGxyODlyOGlzOXp4bXNmdDZiZCZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/3oEduXKKfBX6PPLiGQ/giphy.gif)